tracing/src/main/scala/caliban/tracing/SchemaTracer.scala

```diff
@@ -30,7 +31,7 @@ object SchemaTracer {
         else
           ZIO.serviceWithZIO[Tracing](tracer =>
             tracer.span[R, Nothing, GraphQLResponse[CalibanError]](
-              "query",
+              spanName(request),
```

Comment:
I believe we can achieve the same using combination of an `OverallWrapper` and `EffectfulWrapper` to extract the operation name and populate a Promise or Ref with it. This way we can avoid the breaking change.

You can take a look at how we do something similar in other wrappers such as `ApolloPersistedQueries` and `FieldMetrics`

Reply:
Ah that is a good idea. I will take a look at doing that. My ZIO skills are only moderate, so I may come back here for help but will let you know. The `ApolloPersistedQueries` and `FieldMetrics` are good reference though, thanks for mentioning those.